bootstrap_sqlserver.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# The output of all these installation steps is noisy. With this utility
+# the progress report is nice and concise.
+function install {
+    echo installing $1
+    shift
+    apt-get -y install "$@" >/dev/null 2>&1
+}
+
+# TinyTDS
+install wget wget
+install build-essential build-essential
```

Comment:
No need to install `build-essential` and `wget` again.

Reply:
Removed.